src/SDKs/AnalysisServices/AnalysisServices.Tests/AnalysisServices.Tests.csproj

```diff
@@ -25,6 +25,9 @@
     <None Update="SessionRecords\AnalysisServices.Tests.ScenarioTests.ServerOperationsTests\CreateGetUpdateDeleteTest.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="SessionRecords\AnalysisServices.Tests.ScenarioTests.ServerOperationsTests\CreateServerWithGatewayTest.json">
```

Comment:
@taiwu please remove individual .json files entry and replace it with something like below
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Compute.Tests/Compute.Tests.csproj#L21